testing/tests/index.test.ts

```diff
@@ -1767,6 +1767,41 @@ test("notecards are correctly transformed by the formatNotecards utility", () =>
   );
 });
 
+test("sections should be split by h2, h3 or h4", () => {
+  const builtFolder = path.join(
+    buildRoot,
+    "en-us",
+    "docs",
+    "web",
+    "split_section_by_heading"
+  );
+
+  const jsonFile = path.join(builtFolder, "index.json");
+  const { doc } = JSON.parse(fs.readFileSync(jsonFile));
+  expect(doc.flaws.length).toBeFalsy();
+  expect(doc.title).toBe("Split section by heading");
+
+  const htmlFile = path.join(builtFolder, "index.html");
+  const html = fs.readFileSync(htmlFile, "utf-8");
+  const $ = cheerio.load(html);
+
+  expect($("h2#some_heading").text()).toBe("Some heading");
+  expect($("h2#some_heading").attr("id")).toBe("some_heading");
+  expect($("h2#some_heading").html()).toBe(
```

Comment:
Extract a variable:

```suggestion
  const h2 = $("h2#some_heading");
  expect(h2.text()).toBe("Some heading");
  expect(h2.attr("id")).toBe("some_heading");
  expect(h2.html()).toBe(
```

Reply:
Done! To be more DRY I extracted the h2/h3/h4 selectors to a variable to reduce duplication.